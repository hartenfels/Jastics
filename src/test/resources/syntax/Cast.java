public class Cast knows "music.rdf" {
  public static void test() {
    for («:MusicArtist» artist : do-query(":MusicArtist")) {
      ∃«:influencedBy» ⇒ ⊤ i0 = (∃«:influencedBy» ⇒ ⊤) artist;
      ∃«:influencedBy» ⇒ ⊤ i1 = (∃«:influencedBy» ⇒ ⊤) (artist);
      ∃«:influencedBy» ⇒ ⊤ i2 = ((∃«:influencedBy» ⇒ ⊤) (artist));
    }
  }
}
