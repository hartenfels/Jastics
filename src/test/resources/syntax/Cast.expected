public class Cast from "music.rdf" {
  public static void test() {
    for («:MusicArtist» artist : (query-for(":MusicArtist"))) {
      ∃«:influencedBy»·⊤ i0 = ((∃«:influencedBy»·⊤) artist);
      ∃«:influencedBy»·⊤ i1 = ((∃«:influencedBy»·⊤) (artist));
      ∃«:influencedBy»·⊤ i2 = (((∃«:influencedBy»·⊤) (artist)));
    }
  }
}
